bundles/gcp-bundle/src/main/java/org/apache/gravitino/gcs/fs/GCSFileSystemProvider.java

```diff
@@ -31,12 +33,18 @@
 public class GCSFileSystemProvider implements FileSystemProvider {
   private static final Logger LOGGER = LoggerFactory.getLogger(GCSFileSystemProvider.class);
 
+  public static final Map<String, String> GRAVITINO_KEY_TO_GCS_HADOOP_KEY =
+      ImmutableMap.of(
+          GCSProperties.GCS_SERVICE_ACCOUNT_JSON_PATH, "fs.gs.auth.service.account.json.keyfile");
```

Comment:
Can you please define this "fs.gs.auth.service.account.json.keyfile" as static variable.